docs/docs/configuring-usage-with-plugin-options.md

```diff
@@ -276,6 +276,16 @@ describe(`pluginOptionsSchema`, () => {
 })
 ```
 
+### Opting out of plugin option validation
+
+If you are running your site and run into an error trying to develop or build your site due to plugin options failing that are undocumented or seem erroneous, you can add the `SKIP_PLUGIN_OPTION_VALIDATION` flag set to `true`. It can be prepended to your develop and build scripts, included as an [environment variable](/docs/environment-variables/) in a .env file, or included before your `gatsby develop` command like this:
+
+```shell
+SKIP_PLUGIN_OPTION_VALIDATION=true gatsby develop
```

Comment:
**Bikeshedding, feel free to ignore/postpone**

I'd rather not add this to the docs so explicitly because we don't _really_ want people to use this _unless_ they have a blocking error.

I would prefer if we add that information to the error log if they have an error with an ominous warning, for example something like:

```
success open and validate gatsby-configs - 0.017s

 ERROR #11331  PLUGIN

Invalid plugin options for "gatsby-plugin-manifest":

- "name" must be a string

If you are certain your configuration is correct, you can disable configuration validation with the SKIP_PLUGIN_OPTION_VALIDATION environment variable. Be careful, this could have unintended side effects!

not finished load plugins - 2.391s
```

Reply:
Ooh, this is a fun one. Ya, I totally see what you mean. At the same time, how often do people skip error messages and go googling? No strong feelings, just a weird thought exercise.